client/src/components/TaskDurationBarChart/TaskDurationBarChart.js

```diff
@@ -40,8 +60,28 @@ export const TaskDurationBarChart = ({ taskResponseData }) => {
           left: 20,
           bottom: 5,
         }}
+        barCategoryGap={1000}
```

Comment:
Is this still needed? (i.e. If removed, will this change anything with the way the chart renders?)